### PR TITLE
Refactor - arguments simplified to object

### PR DESCRIPTION
**Pull request by alvaropinot**
I've liked your project, this is the first draft for simplified arguments. If you like like I'll continue working on the other TODOs, as well as adding a defualt config etc :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -249,11 +249,25 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js">
     </script>
     <script type="text/javascript">
-      init('myCanvas',300,true,'#27ae60','#2ecc71');
+      init({
```
Comment (alvaropinot) [inline index.html:252]: updated the demo args

**Comment by flouthoc** [comment]
@alvaropinot  Thanks for the pull request, I'll check it and if it works good for me i'll Merge it asap. :)

**Comment by alvaropinot** [comment]
perfect! shall work. As I've said, if you like I'll prepare another version with a default config object and prepare the project for minification.

**Comment by flouthoc** [comment]
@alvaropinot  there is no need for minification as such, cause code is not that big . actually i copied the whole documentation template from my previous plugin page so "minification came with it" no specific need for that, Yeah default config object would be great you can work on i'll merge it as soon as i'll get time to test your pulls :) Cheers

**Comment by flouthoc** [comment]
@alvaropinot looks cool to me :+1:

**Pull request merged**